controllers/openshift_controller.go

```diff
@@ -218,6 +239,37 @@ func (r *KataConfigOpenShiftReconciler) processDaemonsetForMonitor() *appsv1.Dae
 	}
 }
 
+func (r *KataConfigOpenShiftReconciler) processDashboardConfigMap() *corev1.ConfigMap {
+
+	r.Log.Info("Creating sandboxed containers dashboard in the OpenShift console")
+	cmName := "grafana-dashboard-sandboxed-containers"
+	cmLabels := map[string]string{
+		"console.openshift.io/dashboard": "true",
+	}
+	cmNamespace := "openshift-config-managed"
```

Comment:
@littlejawa can you please add `openshift-config-managed` and `grafana-dashboard-sandboxed-containers` as constants at the top of the file? We need to do that for other such existing strings as well. But lets start with your code ;)

Reply:
Done !